scripts/build_libzmq.sh

```diff
@@ -23,9 +23,9 @@ cd "${ZMQ_SRC_DIR}"
 
 test -f configure || ./autogen.sh
 if [ "$ZMQ" = "4.1.6" ]; then
-  ./configure "--prefix=${ZMQ_PREFIX}" --with-relaxed --enable-static --disable-shared ;
+  ./configure "--prefix=${ZMQ_PREFIX}" --with-relaxed --enable-static --disable-shared --without-documentation ;
 else
-  ./configure "--prefix=${ZMQ_PREFIX}" --disable-pedantic --enable-static --disable-shared ;
+  ./configure "--prefix=${ZMQ_PREFIX}" --disable-pedantic --enable-static --disable-shared --without-documentation ;
```

Comment:
It looks like this option was deprecated in 4.2: https://travis-ci.org/zeromq/zeromq.js/jobs/222752976#L217

We're probably supposed to use `--without-docs` here.